TVRemoteControlUnitTests/LANScannerTests.swift

```diff
@@ -1,7 +1,7 @@
-//@testable import TVRemoteControl
-//import XCTest
+// @testable import TVRemoteControl
```

Comment:
These changes aren't related to the PR. It would be nice if you could revert it.

Reply:
Done.